kolena/workflow/test_sample.py

```diff
@@ -286,7 +287,19 @@ def _data_type(cls) -> _TestSampleType:
         return _TestSampleType.POINT_CLOUD
 
 
-_TEST_SAMPLE_BASE_TYPES = [Composite, Image, Text, BaseVideo, Document, PointCloud]
+@dataclass(frozen=True, config=ValidatorConfig)
+class Audio(TestSample):
+    """An audio file located in a cloud bucket or served at a URL."""
+
+    locator: str
+    """URL (e.g. S3, HTTPS) of the audio file."""
```

Comment:
Are there any restrictions on the format (e.g. `.mp3`)?

Reply:
Will update this with the supported file types.